postfix fails to build with GCC 15 on amd64

The package postfix with version 3.10.2 and epoch 0 fails to build from source on amd64 with the following error:

```
2025-06-02 08:49:39.839921979 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c warn_stat.c
2025-06-02 08:49:39.873443819 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_sockmap.c
2025-06-02 08:49:39.960992215 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c line_number.c
2025-06-02 08:49:39.991173597 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c recv_pass_attr.c
2025-06-02 08:49:40.033712667 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c pass_accept.c
2025-06-02 08:49:40.081233832 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c poll_fd.c
2025-06-02 08:49:40.129545778 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c timecmp.c
2025-06-02 08:49:40.155680031 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_pipe.c
2025-06-02 08:49:40.229885243 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_random.c
2025-06-02 08:49:40.298068876 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c valid_utf8_hostname.c
2025-06-02 08:49:40.330570311 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c midna_domain.c
2025-06-02 08:49:40.411612021 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c argv_splitq.c
2025-06-02 08:49:40.460420657 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c balpar.c
2025-06-02 08:49:40.493761769 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_union.c
2025-06-02 08:49:40.570202114 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c extpar.c
2025-06-02 08:49:40.644136063 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_inline.c
2025-06-02 08:49:40.709820143 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c casefold.c
2025-06-02 08:49:40.768812794 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_utf8.c
2025-06-02 08:49:40.843915952 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c strcasecmp_utf8.c
2025-06-02 08:49:40.883773028 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c split_qnameval.c
2025-06-02 08:49:40.935111372 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c argv_attr_print.c
2025-06-02 08:49:40.98362005 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c argv_attr_scan.c
2025-06-02 08:49:41.033448563 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_file.c
2025-06-02 08:49:41.126237096 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c msg_logger.c
2025-06-02 08:49:41.215964072 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c logwriter.c
2025-06-02 08:49:41.274593527 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c unix_dgram_connect.c
2025-06-02 08:49:41.327187504 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c unix_dgram_listen.c
2025-06-02 08:49:41.381286863 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c byte_mask.c
2025-06-02 08:49:41.457400112 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c known_tcp_ports.c
2025-06-02 08:49:41.509797776 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c argv_split_at.c
2025-06-02 08:49:41.554919785 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_stream.c
2025-06-02 08:49:41.619834356 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c sane_strtol.c
2025-06-02 08:49:41.654726938 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c hash_fnv.c
2025-06-02 08:49:41.70859354 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c ldseed.c
2025-06-02 08:49:41.764940982 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_db.c
2025-06-02 08:49:41.795709843 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_dbm.c
2025-06-02 08:49:41.826572364 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_fail.c
2025-06-02 08:49:41.864970154 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_open.c
2025-06-02 08:49:41.91795963 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c inet_prefix_top.c
2025-06-02 08:49:41.963599397 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c inet_addr_sizes.c
2025-06-02 08:49:42.001263324 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c quote_for_json.c
2025-06-02 08:49:42.058527287 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mystrerror.c
2025-06-02 08:49:42.090034556 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c sane_sockaddr_to_hostaddr.c
2025-06-02 08:49:42.129507176 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c normalize_ws.c
2025-06-02 08:49:42.175913535 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c valid_uri_scheme.c
2025-06-02 08:49:42.227025772 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c clean_ascii_cntrl_space.c
2025-06-02 08:49:42.277107972 +0000 UTC : rv libpostfix-util.so alldig.o allprint.o argv.o argv_split.o attr_clnt.o attr_print0.o attr_print64.o attr_print_plain.o attr_scan0.o attr_scan64.o attr_scan_plain.o auto_clnt.o base64_code.o basename.o binhash.o chroot_uid.o cidr_match.o clean_env.o close_on_exec.o concatenate.o ctable.o dict.o dict_alloc.o dict_cidr.o dict_db.o dict_dbm.o dict_debug.o dict_env.o dict_ht.o dict_ni.o dict_nis.o dict_nisplus.o dict_open.o dict_regexp.o dict_static.o dict_tcp.o dict_unix.o dir_forest.o doze.o dummy_read.o dummy_write.o duplex_pipe.o environ.o events.o exec_command.o fifo_listen.o fifo_trigger.o file_limit.o find_inet.o fsspace.o fullname.o get_domainname.o get_hostname.o hex_code.o hex_quote.o host_port.o htable.o inet_addr_host.o inet_addr_list.o inet_addr_local.o inet_connect.o inet_listen.o inet_proto.o inet_trigger.o line_wrap.o lowercase.o lstat_as.o mac_expand.o load_lib.o mac_parse.o make_dirs.o mask_addr.o match_list.o match_ops.o msg.o msg_output.o msg_syslog.o msg_vstream.o mvect.o myaddrinfo.o myflock.o mymalloc.o myrand.o mystrtok.o name_code.o name_mask.o netstring.o neuter.o non_blocking.o nvtable.o open_as.o open_limit.o open_lock.o peekfd.o posix_signals.o printable.o rand_sleep.o readlline.o ring.o safe_getenv.o safe_open.o sane_accept.o sane_connect.o sane_link.o sane_rename.o sane_socketpair.o sane_time.o scan_dir.o set_eugid.o set_ugid.o sigdelay.o skipblanks.o sock_addr.o spawn_command.o split_at.o split_nameval.o stat_as.o stream_connect.o stream_listen.o stream_recv_fd.o stream_send_fd.o stream_trigger.o sys_compat.o timed_connect.o timed_read.o timed_wait.o timed_write.o translit.o trimblanks.o unescape.o unix_connect.o unix_listen.o unix_recv_fd.o unix_send_fd.o unix_trigger.o unsafe.o uppercase.o username.o valid_hostname.o vbuf.o vbuf_print.o vstream.o vstream_popen.o vstring.o vstring_vstream.o watchdog.o write_buf.o sane_basename.o format_tv.o allspace.o allascii.o load_file.o killme_after.o vstream_tweak.o pass_trigger.o edit_file.o inet_windowsize.o unix_pass_fd_fix.o dict_cache.o valid_utf8_string.o dict_thash.o ip_match.o nbbio.o base32_code.o dict_test.o dict_fail.o msg_rate_delay.o dict_surrogate.o warn_stat.o dict_sockmap.o line_number.o recv_pass_attr.o pass_accept.o poll_fd.o timecmp.o dict_pipe.o dict_random.o valid_utf8_hostname.o midna_domain.o argv_splitq.o balpar.o dict_union.o extpar.o dict_inline.o casefold.o dict_utf8.o strcasecmp_utf8.o split_qnameval.o argv_attr_print.o argv_attr_scan.o dict_file.o msg_logger.o logwriter.o unix_dgram_connect.o unix_dgram_listen.o byte_mask.o known_tcp_ports.o argv_split_at.o dict_stream.o sane_strtol.o hash_fnv.o ldseed.o mkmap_db.o mkmap_dbm.o mkmap_fail.o mkmap_open.o inet_prefix_top.o inet_addr_sizes.o quote_for_json.o mystrerror.o sane_sockaddr_to_hostaddr.o normalize_ws.o valid_uri_scheme.o clean_ascii_cntrl_space.o
2025-06-02 08:49:42.277961234 +0000 UTC : libpostfix-util.so
2025-06-02 08:49:42.27871661 +0000 UTC gcc -shared -Wl,-soname,libpostfix-util.so -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib/postfix -o libpostfix-util.so alldig.o allprint.o argv.o argv_split.o attr_clnt.o attr_print0.o attr_print64.o attr_print_plain.o attr_scan0.o attr_scan64.o attr_scan_plain.o auto_clnt.o base64_code.o basename.o binhash.o chroot_uid.o cidr_match.o clean_env.o close_on_exec.o concatenate.o ctable.o dict.o dict_alloc.o dict_cidr.o dict_db.o dict_dbm.o dict_debug.o dict_env.o dict_ht.o dict_ni.o dict_nis.o dict_nisplus.o dict_open.o dict_regexp.o dict_static.o dict_tcp.o dict_unix.o dir_forest.o doze.o dummy_read.o dummy_write.o duplex_pipe.o environ.o events.o exec_command.o fifo_listen.o fifo_trigger.o file_limit.o find_inet.o fsspace.o fullname.o get_domainname.o get_hostname.o hex_code.o hex_quote.o host_port.o htable.o inet_addr_host.o inet_addr_list.o inet_addr_local.o inet_connect.o inet_listen.o inet_proto.o inet_trigger.o line_wrap.o lowercase.o lstat_as.o mac_expand.o load_lib.o mac_parse.o make_dirs.o mask_addr.o match_list.o match_ops.o msg.o msg_output.o msg_syslog.o msg_vstream.o mvect.o myaddrinfo.o myflock.o mymalloc.o myrand.o mystrtok.o name_code.o name_mask.o netstring.o neuter.o non_blocking.o nvtable.o open_as.o open_limit.o open_lock.o peekfd.o posix_signals.o printable.o rand_sleep.o readlline.o ring.o safe_getenv.o safe_open.o sane_accept.o sane_connect.o sane_link.o sane_rename.o sane_socketpair.o sane_time.o scan_dir.o set_eugid.o set_ugid.o sigdelay.o skipblanks.o sock_addr.o spawn_command.o split_at.o split_nameval.o stat_as.o  stream_connect.o stream_listen.o stream_recv_fd.o stream_send_fd.o stream_trigger.o sys_compat.o timed_connect.o timed_read.o timed_wait.o timed_write.o translit.o trimblanks.o unescape.o unix_connect.o unix_listen.o unix_recv_fd.o unix_send_fd.o unix_trigger.o unsafe.o uppercase.o username.o valid_hostname.o vbuf.o vbuf_print.o vstream.o vstream_popen.o vstring.o vstring_vstream.o watchdog.o write_buf.o sane_basename.o format_tv.o allspace.o allascii.o load_file.o killme_after.o vstream_tweak.o pass_trigger.o edit_file.o inet_windowsize.o unix_pass_fd_fix.o dict_cache.o valid_utf8_string.o dict_thash.o ip_match.o nbbio.o base32_code.o dict_test.o dict_fail.o msg_rate_delay.o dict_surrogate.o warn_stat.o dict_sockmap.o line_number.o recv_pass_attr.o pass_accept.o poll_fd.o timecmp.o  dict_pipe.o dict_random.o valid_utf8_hostname.o midna_domain.o argv_splitq.o balpar.o dict_union.o extpar.o dict_inline.o casefold.o dict_utf8.o strcasecmp_utf8.o split_qnameval.o argv_attr_print.o argv_attr_scan.o dict_file.o msg_logger.o logwriter.o unix_dgram_connect.o unix_dgram_listen.o byte_mask.o known_tcp_ports.o argv_split_at.o dict_stream.o sane_strtol.o hash_fnv.o ldseed.o mkmap_db.o mkmap_dbm.o mkmap_fail.o mkmap_open.o inet_prefix_top.o inet_addr_sizes.o quote_for_json.o mystrerror.o sane_sockaddr_to_hostaddr.o normalize_ws.o valid_uri_scheme.o clean_ascii_cntrl_space.o -lssl -lcrypto -lsasl2 -lnsl -lresolv -ldl -licui18n -licuuc -licudata
2025-06-02 08:49:42.340677838 +0000 UTC cp libpostfix-util.so ../../lib
2025-06-02 08:49:42.342499756 +0000 UTC : ../../lib/libpostfix-util.so
2025-06-02 08:49:42.344059296 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_pcre.c
2025-06-02 08:49:42.560991777 +0000 UTC gcc -shared -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib/postfix -o postfix-pcre.so dict_pcre.o -lpcre2-8
2025-06-02 08:49:42.578520634 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_lmdb.c
2025-06-02 08:49:42.628961045 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_lmdb.c
2025-06-02 08:49:42.821494191 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c slmdb.c
2025-06-02 08:49:43.078190763 +0000 UTC gcc -shared -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib/postfix -o postfix-lmdb.so mkmap_lmdb.o dict_lmdb.o \
2025-06-02 08:49:43.07823553 +0000 UTC     slmdb.o -llmdb
2025-06-02 08:49:43.095840995 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_cdb.c
2025-06-02 08:49:43.184327597 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_cdb.c
2025-06-02 08:49:43.242393984 +0000 UTC gcc -shared -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib/postfix -o postfix-cdb.so mkmap_cdb.o \
2025-06-02 08:49:43.242439548 +0000 UTC     dict_cdb.o 
2025-06-02 08:49:43.253380534 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c mkmap_sdbm.c
2025-06-02 08:49:43.308887764 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -DLINUX6 -c dict_sdbm.c
2025-06-02 08:49:43.38449825 +0000 UTC gcc -shared -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib/postfix -o postfix-sdbm.so mkmap_sdbm.o \
2025-06-02 08:49:43.384523259 +0000 UTC     dict_sdbm.o 
2025-06-02 08:49:43.427978404 +0000 UTC for i in postfix-pcre.so postfix-lmdb.so postfix-cdb.so postfix-sdbm.so; \
2025-06-02 08:49:43.42799312 +0000 UTC do \
2025-06-02 08:49:43.427999938 +0000 UTC   for type in shl_load pcre pgsql mysql ldap sqlite lmdb dev_urandom; do \
2025-06-02 08:49:43.428005692 +0000 UTC       case $i in postfix-$type.so) \
2025-06-02 08:49:43.428011456 +0000 UTC 	  cmp -s $i ../../lib/$i 2>/dev/null || cp $i ../../lib; \
2025-06-02 08:49:43.428016897 +0000 UTC 	  continue 2;; \
2025-06-02 08:49:43.428021998 +0000 UTC       esac; \
2025-06-02 08:49:43.428034728 +0000 UTC   done; \
2025-06-02 08:49:43.428039962 +0000 UTC   rm -f ../../lib/$i; \
2025-06-02 08:49:43.428048476 +0000 UTC done
2025-06-02 08:49:43.437153011 +0000 UTC for i in ; \
2025-06-02 08:49:43.437164306 +0000 UTC do \
2025-06-02 08:49:43.43716856 +0000 UTC   cmp -s $i ../../lib/$i 2>/dev/null || cp $i ../../lib; \
2025-06-02 08:49:43.437171926 +0000 UTC done
2025-06-02 08:49:43.43808346 +0000 UTC for i in argv.h attr.h attr_clnt.h auto_clnt.h base64_code.h binhash.h chroot_uid.h cidr_match.h clean_env.h connect.h ctable.h dict.h dict_cdb.h dict_cidr.h dict_db.h dict_dbm.h dict_env.h dict_ht.h dict_lmdb.h dict_ni.h dict_nis.h dict_nisplus.h dict_pcre.h dict_regexp.h dict_sdbm.h dict_static.h dict_tcp.h dict_unix.h dir_forest.h events.h exec_command.h find_inet.h fsspace.h fullname.h get_domainname.h get_hostname.h hex_code.h hex_quote.h host_port.h htable.h inet_addr_host.h inet_addr_list.h inet_addr_local.h inet_proto.h iostuff.h line_wrap.h listen.h lstat_as.h mac_expand.h mac_parse.h make_dirs.h mask_addr.h match_list.h msg.h msg_output.h msg_syslog.h msg_vstream.h mvect.h myaddrinfo.h myflock.h mymalloc.h myrand.h name_code.h name_mask.h netstring.h nvtable.h open_as.h open_lock.h posix_signals.h readlline.h ring.h safe.h safe_open.h sane_accept.h sane_connect.h sane_fsops.h load_lib.h sane_socketpair.h sane_time.h scan_dir.h set_eugid.h set_ugid.h sigdelay.h sock_addr.h spawn_command.h split_at.h stat_as.h stringops.h sys_defs.h timed_connect.h timed_wait.h trigger.h username.h valid_hostname.h vbuf.h vbuf_print.h vstream.h vstring.h vstring_vstream.h watchdog.h format_tv.h load_file.h killme_after.h edit_file.h dict_cache.h dict_thash.h ip_match.h nbbio.h base32_code.h dict_fail.h warn_stat.h dict_sockmap.h line_number.h timecmp.h slmdb.h compat_va_copy.h dict_pipe.h dict_random.h valid_utf8_hostname.h midna_domain.h dict_union.h dict_inline.h check_arg.h argv_attr.h msg_logger.h logwriter.h byte_mask.h known_tcp_ports.h sane_strtol.h hash_fnv.h ldseed.h mkmap.h inet_prefix_top.h inet_addr_sizes.h valid_uri_scheme.h clean_ascii_cntrl_space.h; \
2025-06-02 08:49:43.438093824 +0000 UTC do \
2025-06-02 08:49:43.438098057 +0000 UTC   cmp -s $i ../../include/$i 2>/dev/null || cp $i ../../include; \
2025-06-02 08:49:43.438101738 +0000 UTC done
2025-06-02 08:49:43.828138186 +0000 UTC cd ../../include; chmod 644 argv.h attr.h attr_clnt.h auto_clnt.h base64_code.h binhash.h chroot_uid.h cidr_match.h clean_env.h connect.h ctable.h dict.h dict_cdb.h dict_cidr.h dict_db.h dict_dbm.h dict_env.h dict_ht.h dict_lmdb.h dict_ni.h dict_nis.h dict_nisplus.h dict_pcre.h dict_regexp.h dict_sdbm.h dict_static.h dict_tcp.h dict_unix.h dir_forest.h events.h exec_command.h find_inet.h fsspace.h fullname.h get_domainname.h get_hostname.h hex_code.h hex_quote.h host_port.h htable.h inet_addr_host.h inet_addr_list.h inet_addr_local.h inet_proto.h iostuff.h line_wrap.h listen.h lstat_as.h mac_expand.h mac_parse.h make_dirs.h mask_addr.h match_list.h msg.h msg_output.h msg_syslog.h msg_vstream.h mvect.h myaddrinfo.h myflock.h mymalloc.h myrand.h name_code.h name_mask.h netstring.h nvtable.h open_as.h open_lock.h posix_signals.h readlline.h ring.h safe.h safe_open.h sane_accept.h sane_connect.h sane_fsops.h load_lib.h sane_socketpair.h sane_time.h scan_dir.h set_eugid.h set_ugid.h sigdelay.h sock_addr.h spawn_command.h split_at.h stat_as.h stringops.h sys_defs.h timed_connect.h timed_wait.h trigger.h username.h valid_hostname.h vbuf.h vbuf_print.h vstream.h vstring.h vstring_vstream.h watchdog.h format_tv.h load_file.h killme_after.h edit_file.h dict_cache.h dict_thash.h ip_match.h nbbio.h base32_code.h dict_fail.h warn_stat.h dict_sockmap.h line_number.h timecmp.h slmdb.h compat_va_copy.h dict_pipe.h dict_random.h valid_utf8_hostname.h midna_domain.h dict_union.h dict_inline.h check_arg.h argv_attr.h msg_logger.h logwriter.h byte_mask.h known_tcp_ports.h sane_strtol.h hash_fnv.h ldseed.h mkmap.h inet_prefix_top.h inet_addr_sizes.h valid_uri_scheme.h clean_ascii_cntrl_space.h
2025-06-02 08:49:43.830964296 +0000 UTC [src/global]
2025-06-02 08:49:43.836111776 +0000 UTC gcc -I. -I../../include -DHAS_SHL_LOAD -DNO_DB -DHAS_PCRE=2 -DUSE_TLS -DUSE_SASL_AUTH -DDEF_SASL_SERVER="dovecot" -DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl -DHAS_PGSQL -DHAS_MYSQL -I/usr/include/mysql -I/usr/include/mysql/mysql -DHAS_LDAP -DUSE_LDAP_SASL -DHAS_SQLITE -DHAS_LMDB -DDEF_DB_TYPE="lmdb" -DHAS_DEV_URANDOM -DDEF_CONFIG_DIR="/etc/postfix" -DDEF_DAEMON_DIR="/usr/libexec/postfix" -DDEF_META_DIR="/etc/postfix" -DDEF_SENDMAIL_PATH="/usr/bin/sendmail" -DDEF_SHLIB_DIR="/usr/lib/postfix" -DUSE_DYNAMIC_LIBS -DUSE_DYNAMIC_MAPS -Wmissing-prototypes -Wformat -Wno-comment -fno-common -fPIC   -I. -I../../include -DLINUX6 -c abounce.c
2025-06-02 08:49:43.862358514 +0000 UTC In file included from abounce.c:195:
2025-06-02 08:49:43.862402433 +0000 UTC ./mail_params.h:17:13: error: 'bool' cannot be defined via 'typedef'
2025-06-02 08:49:43.862408931 +0000 UTC    17 | typedef int bool;
2025-06-02 08:49:43.862413024 +0000 UTC       |             ^~~~
2025-06-02 08:49:43.862416757 +0000 UTC ./mail_params.h:17:13: note: 'bool' is a keyword with '-std=c23' onwards
2025-06-02 08:49:43.86242029 +0000 UTC ./mail_params.h:17:1: warning: useless type name in empty declaration
2025-06-02 08:49:43.862423694 +0000 UTC    17 | typedef int bool;
2025-06-02 08:49:43.862426958 +0000 UTC       | ^~~~~~~
2025-06-02 08:49:43.906785386 +0000 UTC make: *** [Makefile:207: abounce.o] Error 1
2025-06-02 08:49:43.907226919 +0000 UTC make: *** [Makefile:112: update] Error 1
2025-06-02 08:49:44.538242896 +0000 UTC failed to build package: unable to run package postfix pipeline: unable to run pipeline: exit status 2
2025-06-02 08:49:44.626426869 +0000 UTC failed to build: command melange failed: exit status 1
2025-06-02 08:49:44.626599789 +0000 UTC build failed: command melange failed: exit status 1
```

--- a/postfix.yaml
+++ b/postfix.yaml
@@ -1,7 +1,7 @@
 package:
   name: postfix
   version: "3.10.2"
-  epoch: 2
+  epoch: 3
   description: Secure and fast drop-in replacement for Sendmail (MTA)
   copyright:
     - license: IPL-1.0 OR EPL-2.0
@@ -28,7 +28,6 @@ environment:
       - ca-certificates-bundle
       - coreutils
       - cyrus-sasl-dev
-      - gcc-14-default
       - icu-dev
       - libnsl-dev
       - linux-headers
@@ -112,6 +111,10 @@ pipeline:
 
       # Add LMDB settings and make it default
       ccargs="$ccargs -DHAS_LMDB $(pkg-config --cflags lmdb) -DDEF_DB_TYPE=\\\"lmdb\\\""
+
+      # Fix build error with gcc 15.
+      # TODO: remove once postfix 3.11 released
+      ccargs="$ccargs -std=gnu17"
 
       mkdir -p "${{targets.contextdir}}"/usr/bin
       mkdir -p "${{targets.contextdir}}"/etc/postfix

